### PR TITLE
M0078DDG-431 update several glossary entries

### PR DESCRIPTION
**Pull request by fpinel-microej**

### CODE REVIEW

File: glossary.rst [hunk 1]
```diff
@@ -64,6 +64,9 @@ This glossary defines the technical terms upon which the `MicroEJ Virtual Execut
    Studio
       MicroEJ Studio allows application developers to write a MicroEJ Sandboxed Application, run it on a Virtual Device, deploy it on a MicroEJ-ready device, and publish it to a `MicroEJ Forge <https://www.microej.com/product/forge/>`_ instance.
 
+   VEE
+   MICROEJ VEE is an applications container. VEE stands for Virtual Execution Environment, and refers to the first implementation that embeds a virtual 32-bit processor, so the term "Virtual". MICROEJ VEE runs on any OS/RTOS commonly used in embedded systems (FreeRTOS, QP/C, uc/OS, ThreadX, embOS, Mbed OS, Zephyr OS, VxWorks, PikeOS, Integrity, Linux, QNX, …) and can also run without RTOS (bare-metal) or proprietary RTOS. MICROEJ VEE includes the small MEJ32, along with a wide range of libraries (Add-On Libraries and Foundation Libraries).
```
Comment (fpinel-microej) [inline glossary.rst:68]: "so the term "Virtual"" ==> "hence the term "Virtual"" ?

File: glossary.rst [hunk 2]
```diff
@@ -53,7 +53,7 @@ This glossary defines the technical terms upon which the `MicroEJ Virtual Execut
       MicroEJ Module Manager downloads, installs and controls the consistency of all the dependencies and versions required to build and publish a MicroEJ asset. It is based on `Semantic Versioning <https://semver.org>`_ specification.
 
    Platform
-      A MicroEJ Platform integrates a MicroEJ Architecture, one or more Foundation Libraries with their respective Abstraction Layers and the board support package (BSP) for the target Device. It also includes associated MicroEJ Mocks for the MicroEJ Simulator.
+      A MicroEJ Platform integrates a MICROEJ VEE, a MICROEJ Architecture, one or more Foundation Libraries with their respective Abstraction Layers, and the board support package (BSP) for the target Device. It also includes associated MICROEJ Mocks for the MICROEJ Simulator.
```
Comment (fpinel-microej) [inline glossary.rst:56]: Should "MicroEJ" be kept in all caps or shoud it be corrected ? I'm guessing we should keep the former orthograph to match the rest of the glossary but I prefer asking before changing.

File: glossary.rst [hunk 3]
```diff
@@ -31,8 +31,8 @@ This glossary defines the technical terms upon which the `MicroEJ Virtual Execut
    Architecture
       A MicroEJ Architecture is a software package that includes the MicroEJ Core Engine port to a target instruction set and a C compiler, core MicroEJ Foundation Libraries (EDC, :ref:`[BON], [SNI], [KF] <esr-specifications>`) and the MicroEJ Simulator. MicroEJ Architectures are distributed either as evaluation or production version.
 
-   Core Engine
-      MicroEJ Core Engine is a scalable runtime for resource-constrained embedded devices running on 32-bit microcontrollers or microprocessors. MicroEJ Core Engine allows devices to run multiple and mixed Java and C software applications.
+   Core Engine, also named "MEJ32"
```
Comment (fpinel-microej) [inline glossary.rst:34]: suggesting "Core Engine (also named "MEJ32")"

File: glossary.rst [hunk 4]
```diff
@@ -64,6 +64,9 @@ This glossary defines the technical terms upon which the `MicroEJ Virtual Execut
    Studio
       MicroEJ Studio allows application developers to write a MicroEJ Sandboxed Application, run it on a Virtual Device, deploy it on a MicroEJ-ready device, and publish it to a `MicroEJ Forge <https://www.microej.com/product/forge/>`_ instance.
 
+   VEE
+      MICROEJ VEE is an applications container. VEE stands for Virtual Execution Environment, and refers to the first implementation that embeds a virtual 32-bit processor, so the term "Virtual". MICROEJ VEE runs on any OS/RTOS commonly used in embedded systems (FreeRTOS, QP/C, uc/OS, ThreadX, embOS, Mbed OS, Zephyr OS, VxWorks, PikeOS, Integrity, Linux, QNX, …) and can also run without RTOS (bare-metal) or proprietary RTOS. MICROEJ VEE includes the small MEJ32, along with a wide range of libraries (Add-On Libraries and Foundation Libraries).
```
Comment (fpinel-microej) [inline glossary.rst:68]: "so the term "Virtual"" ==> "hence the term "Virtual"" ?

File: glossary.rst [hunk 5]
```diff
@@ -3,7 +3,7 @@
 MicroEJ Glossary
 ================
 
-This glossary defines the technical terms upon which the `MicroEJ Virtual Execution Environment <https://developer.microej.com/microej-vee-virtual-execution-environment>`_ is built.
+This glossary defines the technical terms upon which the `MicroEJ VEE (Virtual Execution Environment) <https://developer.microej.com/microej-vee-virtual-execution-environment>`_ is built.
```
Comment (jerome-leroux) [inline glossary.rst:6]:
```suggestion
This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual Execution Environment) <https://developer.microej.com/microej-vee-virtual-execution-environment>`_ is built.
```

File: glossary.rst (hunk 2 above)
Comment (jerome-leroux) [reply glossary.rst:56]: The products (SDK, Studio, VEE, ...) must be prefixed with "MICROEJ". For the other usages, we can keep "MicroEJ".
We will do a complete review later.

File: glossary.rst (hunk 3 above)
Comment (jerome-leroux) [reply glossary.rst:34]: Let's keep the existing sentence.

File: glossary.rst (hunk 4 above)
Comment (jerome-leroux) [reply glossary.rst:68]:
```suggestion
      MICROEJ VEE is an applications container. VEE stands for Virtual Execution Environment, and refers to the first implementation that embeds a virtual 32-bit processor, hence the term "Virtual". MICROEJ VEE runs on any OS/RTOS commonly used in embedded systems (FreeRTOS, QP/C, uc/OS, ThreadX, embOS, Mbed OS, Zephyr OS, VxWorks, PikeOS, Integrity, Linux, QNX, …) and can also run without RTOS (bare-metal) or proprietary RTOS. MICROEJ VEE includes the small MEJ32, along with a wide range of libraries (Add-On Libraries and Foundation Libraries).
```

**Pull request merged**